Sources/PointFree/Session.swift

```diff
@@ -13,23 +13,24 @@ public enum CookieTransform: String, Codable {
   case encrypted
 }
 
+private let cookieExpirationDuration: TimeInterval = 315_360_000 // 60 * 60 * 24 * 365 * 10
+
 public func writeSessionCookieMiddleware<A>(_ update: @escaping (Session) -> Session)
   -> (Conn<HeadersOpen, A>)
   -> IO<Conn<HeadersOpen, A>> {
-
+    
     return { conn in
       let value = update(conn.request.session)
       guard value != conn.request.session else { return pure(conn) }
-      return setCookie(
```

Comment:
this was taking about 2.5 seconds to compile, and at first I thought it was due to the `map` and `??` stuff, but most likely it was due to the arithmetic being done.

Reply:
Ha jeez, yeah.